doc/download.rst

```diff
@@ -25,35 +25,21 @@ If you use NEST for your project, don't forget to :doc:`cite NEST <citing-nest>`
 Download the source code
 -------------------------
 
-* If you are compiling NEST from source, you can |link-pre|\ |version|\ |link-post|
-
-.. |link-pre| raw:: html
-
-    <a href="https://github.com/nest/nest-simulator/archive/v
-
-.. |link-post| raw:: html
-
-    .tar.gz">download the latest release here</a>
+* If you are compiling NEST from source, you can download the latest release `here <https://github.com/nest/nest-simulator/archive/v2.20.0.tar.gz>`_.
```

Comment:
I assume there was a good reason for @jessica-mitchell to introduce this more complex formatting in #1372. Can you please double-check that your simplification does not break anything?

Reply:
I restored the code and changed a little bit so that the git-link works correctly.